<launch>
  <node pkg="dyret_controller" name="gaitController" type="gaitController" />
  <node pkg="dyret_controller" name="gaitEvaluator" type="gaitEvaluator" />
  <node pkg="dyret_controller" name="inverseKinematicsAdapter" type="inverseKinematicsAdapter" />
</launch>
